src/main/steps/you-can-use-english-welsh-courts/content.ts

```diff
@@ -214,7 +214,13 @@ const cy = ({ isDivorce, partner }: CommonContent, connections: JurisdictionConn
 };
 
 export const form: FormContent = {
-  fields: {},
+  fields: {
+    connections: {
+      type: 'hidden',
+      label: l => l.title,
```

Comment:
i assume this is only for the check your answers page to display a title?

Reply:
Yeah, this input is be hidden on this page, but will be shown with a title on the check your answers page.